src/grunt/helper.js

```diff
@@ -111,34 +111,41 @@ ns.init = function (grunt) {
                 reporters: ['progress', 'coverage'],
 
                 coverageReporter: {
-                    reporters: [{
-                        type: 'text-summary'
-                    }, {
-                        type: 'html',
-                        dir: path.join(process.cwd(), 'coverage'),
-                    }],
+                    reporters: [
+                        {
+                            type: 'text-summary',
+                        },
+                        {
+                            type: 'html',
+                            dir: path.join(process.cwd(), 'coverage'),
+                        },
+                    ],
```

Comment:
While I think hygienically-speaking trailing-commas are nice, I have never thought of them as a "must-have," especially since it's an error in JSON. OTOH, I certainly value vertical space in the editor, so I would therefore vote for shorter-format over enforced trailing-commas.